src/utils/loader.js

```diff
@@ -0,0 +1,7 @@
+import {sampleSize} from 'lodash';
+
+export async function load(count = 10) {
```

Comment:
Since this file exports a single function I would rather export it as default 

```suggestion
export default async function load(count = 10) {
```

Reply:
done :)